consensus/enclave/build.rs

```diff
@@ -29,6 +30,14 @@ fn main() {
         SGX_LIBS
     };
 
+    // This is a hack for lint to run, this wouldn't be necessary after we move the
+    // enclave to its own crate
+    rerun_if_env_changed!("CONSENSUS_ENCLAVE_CSS");
+    if let Ok(value) = var("CONSENSUS_ENCLAVE_CSS") {
+        warning!("Found consensus enclave css: {}", value);
+        return;
+    }
```

Comment:
Are there conditions in which this could introduce side effects?

Reply:
Unfortunately this will break builds of consensus with provided CSS files, because the generated code that lives in this file will not be created (which is required).